report/Introduction/ProjectProposal_eng.tex

```diff
@@ -52,3 +55,12 @@ \subsection{Expected contributions from the project}
     \item How can the properties of a Finite State Automata help the customer make inquiries and innovation on requirements that might not be complete?
     \item How can automatic acceptance tests and precisely defined requirements help developers deliver high quality within a reasonable time period?
 \end{itemize}
+\section{Modifications}
+Due to the Covid-19 situation, the project proposal we would have liked to work with, was not an option.
+Everything that required us to work with the Giraf project groups went down the drain when Aalborg University sent all students home to isolate.
+This also meant that usability testing our product on the groups no longer was an option. 
+
+However we believe that our product proposal is still valid.
+And due to the pandemic situation, now more then before since our product will allow developers and "Product Owner" to be isolated
+
+This means that even though we can no longer test our product in a suited environment, we still believe that the product will have a huge potential to provide a solution, that could allow people to continue to work during in a pandemic like we have seen during this project.
```

Comment:
```suggestion
This means that even though we can no longer test our product in a suited environment, we still believe that the product will have a huge potential to provide a solution, that could allow people to continue to work during in a pandemic like we have seen during this project.
Therefore we believe that a solution which we propuse, is more needed than before the Covid-19 situation, because this might not be the only one.
Furthermore, this solution, could also improve the way that Product Owners and developers work together.
```

Kan godt være at sidste linje ikke skal være med:
```suggestion
This means that even though we can no longer test our product in a suited environment, we still believe that the product will have a huge potential to provide a solution, that could allow people to continue to work during in a pandemic like we have seen during this project.
Therefore we believe that a solution which we propuse, is more needed than before the Covid-19 situation, because this might not be the only one.
```

Reply:
Jeg syntes ikke den er nødvendig, jeg tror det er et lidt bold statement